### DISCUSSION TITLE
Togglable Features

### DISCUSSION THREAD

**Discussion by yacinehamza111** [ppy/osu#26798] [Ideas]
Togglable Features can be a cool feature that if someone doesn't like a feature, he/she can toggle that feature. It can be any feature except the main feature (the togglable features)

**Comment by Stoppedpuma** [comment]
There's already enough toggles, I'm sure @bdach has enough nightmares from maintaining them as is. Most likely wont happen as he already stated to you in Twitch chat last week.

**Comment by yacinehamza111** [comment]
Oh, alright then.
